### PR TITLE
metis 5.2.1

### PR DESCRIPTION
**Pull request by SMillerDev**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by SMillerDev** [comment]
The log mentions something about RPATH, maybe that's why it's failing on all macOS versions?

File: Formula/metis.rb [hunk 1]
```diff
@@ -27,8 +22,18 @@ class Metis < Formula
...
+    url "https://github.com/KarypisLab/GKLib.git", revision: "8bd6bad750b2b0d90800c632cf18e8ee93ad72d7"
+  end
+
   def install
-    system "make", "config", "prefix=#{prefix}", "shared=1"
+    args = %W[prefix=#{prefix} shared=1 cc=#{ENV.cc}]
+    resource("gklib").stage do
+      system "make", "config", *args
+      system "make", "install"
+    end
```
Comment (carlocab) [inline Formula/metis.rb:34]: We should just build this as a static library so we only need to install it somewhere in `buildpath` I think. Unless `metis` headers also need `gklib` headers?

File: Formula/metis.rb [hunk 2]
```diff
@@ -27,8 +22,18 @@ class Metis < Formula
...
+
+    system "make", "config", *args, "gklib_path=#{prefix}"
```
Comment (carlocab) [inline Formula/metis.rb:36]: I suggest trying
```ruby
ENV.append "LDFLAGS", "-Wl,-undefined,dynamic_lookup" if OS.mac?
```
to deal with the build failure

**Comment by MikeMcQuaid** [comment]
Have removed `help wanted` and/or `in progress` from this PR as there's been no progress in a few months. Maintainers: please don't add these labels as the stalebot is performing a useful service in continuing to bump these PRs and closing them if there's no activity after those bumps. Thanks ❤️

**Comment by chenrui333** [comment]
I am gonna close this PR for now, for whoever watching this, feel free to open a new one. :)

**Pull request closed without merging**